packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -256,9 +256,12 @@ async function getAccountIncomingPaymentsPage(
 ): Promise<IncomingPayment[]> {
   assert.ok(deps.knex, 'Knex undefined')
 
-  return await IncomingPayment.query(deps.knex).getPage(pagination).where({
-    accountId: accountId
-  })
+  return await IncomingPayment.query(deps.knex)
+    .getPage(pagination)
+    .where({
+      accountId: accountId
+    })
+    .withGraphFetched('asset')
```

Comment:
Why does that not also need to be `.withGraphFetched('[account.asset, asset]')`?

Reply:
I know I'd previously said otherwise, but I think incoming payment queries really only need `.withGraphFetched('asset')`. The payment's asset should just be checked against the account asset on creation.
`IncomingPayment`'s `account` could be made optional or just removed:
https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/payment/incoming/model.ts#L51